topics/single-cell/tutorials/scrna-data-ingest/tutorial.md

```diff
@@ -0,0 +1,881 @@
+> 
+{: .warning}
+
+## Seurat -> AnnData 
+
+An example of SCEasy Tool in action is shown below. As mentioned above, there are two SCEasy tools currently available. When we speak about conversion to AnnData, the latest tool {% tool [SCEasy Converter](toolshed.g2.bx.psu.edu/repos/iuc/sceasy_convert/sceasy_convert/0.0.7+galaxy2) %} converts Seurat into the latest version of AnnData object. However, currently it is not possible to use that output with downstream EBI single-cell tools because they only support older versions of the AnnData. The solution is to use the second SCEasy tool {% tool [SCEasy convert](toolshed.g2.bx.psu.edu/repos/ebi-gxa/sceasy_convert/sceasy_convert/0.0.5+galaxy1) %} which has limited conversion options but can generate AnnData files compatible with the EBI single-cell tools that we are going to use next in the workflow. 
+
+Our Seurat starting file was generated by pulling data from [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home) and transforming it into the desired format, what was shown in the [data import tutorial]({% link topics/single-cell/tutorials/EBI-retrieval/tutorial.md %}). It will be our toy dataset.
+
+> <hands-on-title>Get toy data</hands-on-title>
+>
+> 1. Create a new history for this tutorial 
+> 2. Import the AnnData object from [Zenodo](https://zenodo.org/records/10397653)
+>
+>    ```
+>    https://zenodo.org/records/10397653/files/Seurat_object.rdata
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    
+> 3. Alternatively, you can import history where we created the Seurat object: [Input history](https://singlecell.usegalaxy.eu/u/j.jakiela/h/ebi-scxa---anndata-scanpy-or-seurat-object-1) 
+>
+>    {% snippet faqs/galaxy/histories_import.md %}
+>
+>
+> 4. **Rename** {% icon galaxy-pencil %} the dataset `Seurat object`
+>
+{: .hands_on}
+
+
+> <hands-on-title> Seurat to AnnData with SCEasy </hands-on-title>
+>
+> 1. {% tool [SCEasy convert](toolshed.g2.bx.psu.edu/repos/ebi-gxa/sceasy_convert/sceasy_convert/0.0.5+galaxy1) %} with the following parameters:
+>    - *"Direction of conversion"*: `Seurat to AnnData`
+>    - {% icon param-file %} *"Input object in Seurat RDS format"*: `Seurat object`
+>      (if the dataset does not show up in the corresponding input field, just drag the dataset from the history panel and drop into the input field)
+>    - *"Name of the assay to be transferred"*: `RNA`
+>    - *"Data type of the assay to be transferred"*: `data`
+>   
+> 2. **Rename** {% icon galaxy-pencil %} the output `Converted EBI-compatible AnnData file`.
+>
+> 3. Check that the datatype is `h5ad`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="h5ad" %}
+>    
+{: .hands_on}
+
+And that's it! The latest tool {% tool [SCEasy Converter](toolshed.g2.bx.psu.edu/repos/iuc/sceasy_convert/sceasy_convert/0.0.7+galaxy2) %} will do the same but the output file will be the newest AnnData version and will not work with the tool used below.  
```

Comment:
Maybe good to have an alternative hands-on with the latest SCEasy converter tool too.

Reply:
I just mentioned the latest SCEasy here without a hands-on box because I thought it's similar enough to the old one that the users will know how to use it. And it would produce the AnnData output which wouldn't be compatible with the Filter, Plot, Explore workflow and in this section the main aim was to do the conversion and pass on the converted object into that workflow. As you pointed out previously, we need an older version of AnnData for that workflow, hence there it is the older version of SCEasy used here.